application/Espo/Modules/Postgres/Core/Utils/Database/Orm/Fields/PersonName.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Espo\Modules\Postgres\Core\Utils\Database\Orm\Fields;
+
+class PersonName extends \Espo\Core\Utils\Database\Orm\Fields\PersonName
+{
+    protected function getSelect(array $fullList, ?string $alias = null): string
+    {
+        foreach ($fullList as &$item) {
+            $rowItem = trim($item, " '");
+
+            if (empty($rowItem)) {
+                continue;
+            }
+
+            if ($alias) {
+                $item = $alias . '.' . $item;
+            }
+
+            $item = "COALESCE:({$item}, '')"; // HERE - change IFNULL to COALESCE, it can probably be done in a better way, not sure
```

Comment:
As NULLIF is a part of Espo and is documented changing it here is not enough. It's needed to be converted on ORM level.

Reply:
I thought so. But it's not NULLIF, it's IFNULL.. Pg actually has NULLIF.